samples/ORKCatalog/ORKCatalog/Resources/Localized/Base.lproj/LaunchScreen.xib

```diff
@@ -25,10 +28,12 @@
                 </navigationBar>
                 <tabBar contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="pMQ-9U-JXX">
                     <rect key="frame" x="0.0" y="431" width="480" height="49"/>
+                    <animations/>
                     <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
                     <items/>
                 </tabBar>
             </subviews>
+            <animations/>
             <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="calibratedWhite"/>
             <constraints>
                 <constraint firstItem="pv7-RZ-bDW" firstAttribute="top" secondItem="KFe-cA-LWE" secondAttribute="bottom" id="ERQ-ec-2pG"/>
```

Comment:
No - will try to revert. Didn't change anything - would Xcode 7.1 Beta have done this (noting the change in toolsVersion)?